airbyte-webapp/src/components/DocumentationPanel/DocumentationPanel.tsx

```diff
@@ -64,24 +61,11 @@ export const DocumentationPanel: React.FC = () => {
     setDocumentationPanelOpen(false);
   }, [setDocumentationPanelOpen, location.pathname]);
 
-  return isLoading ? (
+  return isLoading || documentationUrl === "" ? (
```

Comment:
There's a weird, brief moment when sometimes `documentationUrl` gets set to an empty string (not every time, but sometimes it will be correct for 1-2 triggers, then an empty string, then recorrect).  The result was a brief flash of the "No docs found" screen.  I was unable to track down which hook was causing the issue so I opted for this.  If it seems brittle I'd love to pair on it to come up with a more exact solution!